bindings/python/stp/library_path.py.in

```diff
@@ -27,6 +27,15 @@
 # a good idea.
 
 # Search paths for the stp shared library
-PATHS = [
-    '@CMAKE_LIBRARY_OUTPUT_DIRECTORY@/@LIBSTP_FILENAME@'
+
+import os
+import sys
+
+path_var = 'PATH' if sys.platform in ('win32', 'cygwin') else 'DYLD_LIBRARY_PATH' if sys.platform == 'darwin' else 'LD_LIBRARY_PATH'
```

Comment:
Can you rewrite this to use 80 columns? You could use a dict to map them, or just a bunch of `if ... elif`. Python's ternary operator is so verbose, blech.

Reply:
Done!